jakarta-eclipse/org.eclipse.lsp4jakarta.core/src/org/jakarta/codeAction/proposal/RemoveAnnotationConflictQuickFix.java

```diff
@@ -11,7 +11,7 @@
 * Contributors:
 *     Red Hat Inc. - initial API and implementation
 *******************************************************************************/
-package org.jakarta.jdt.persistence;
+package org.jakarta.codeAction.proposal;
```

Comment:
Can we move this class into a package under `proposal`

`package org.jakarta.codeAction.proposal.quickfix;`

Can we also move [InsertAnnotationMissingQuickFix](https://github.com/eclipse/lsp4jakarta/blob/master/jakarta-eclipse/org.eclipse.lsp4jakarta.core/src/org/jakarta/jdt/servlet/InsertAnnotationMissingQuickFix.java) to this new package as well.